app/views/krikri/harvest_sources/index.html.erb

```diff
@@ -21,5 +26,6 @@
 
 <%= link_to 'New',
     new_institution_harvest_source_path(institution_id: @institution.id)
-%>
+%> | 
```

Comment:
Yes. And probably. :) I can consult with @AudreyAltman and push up a fix. (Alternately I can break out this out into a separate PR.)